app/theme/client/imports/general/base_old.css

```diff
@@ -1858,7 +1858,7 @@ rc-old select,
 
 			overflow: hidden;
 
-			width: 50%;
+			width: 85%;
```

Comment:
what do you think about instead of use width use 
```
.upload-progress-text {

   flex-grow: 1;
   white-space: nowrap;
}
```


and 

```
.rc-old .container-bars .upload-progress {
   display: flex;
}
```

Reply:
Done